lib/reply.js

```diff
@@ -76,6 +84,11 @@ Reply.prototype.code = function (code) {
   return this
 }
 
+Reply.prototype.serializer = function (fn) {
+  this._serializer = fn
+  return this
+}
```

Comment:
As you can see [here](https://github.com/fastify/fastify/blob/6adad7d91684249d5f208b12fb6e0b33b20f6c96/lib/reply.js#L58) and [here](https://github.com/fastify/fastify/blob/6adad7d91684249d5f208b12fb6e0b33b20f6c96/lib/reply.js#L67), we are assuming that the user knows which `content-type` he is using.
Probably we must add this to the docs.